exercises/colorsquare.js

```diff
@@ -54,7 +64,15 @@ class ColorSquare{
 			make sure the rightNeighbor is something!  the rightmost element won't have a neighbor
 		*/
 	handleClick(){
-
+		if (this.colorArray[this.colorIndex + 1]){
+			this.colorIndex += 1;
+		}else if (this.colorArray.length){
+			this.colorIndex = 0;
+		}
+		this.element.css('background-color', this.colorArray[this.colorIndex]);
```

Comment:
if you're going to do this, and the same code in changeColor, why not use the function changeColor?